thirdparty/gtest/googletest/generated/GTestConfig.cmake

```diff
@@ -1,33 +1,33 @@
-
```

Comment:
where the gtests intentional changes?
We might need to add some stuff to .gitignore

Reply:
The changes seem to leave the files identical so I wonder if it there are some vs shenanigans going on

Reply:
Never touched the cmake so probably